sigma/plugins/utility/other/define.py

```diff
@@ -0,0 +1,72 @@
+import discord
+import aiohttp
+from bs4 import BeautifulSoup
+
+async def define(cmd, message, args):
+
+    definition = ""
+    defNum = 1
+    defLast = False
+    hasPrinted = False
+    defURL = r"https://www.merriam-webster.com/dictionary/"
+
+    if not args:
+        await message.channel.send(cmd.help())
+        return
+    try:
+        defNum = int(args[-1])
+        defLast = True
+    except ValueError:
+        pass
+
+    if not defLast:
+        for i in args:
+            defURL += i
+            defURL += "%20"
+    else:
+        argsCopy = args.copy()
+        del(argsCopy[-1])
+        for i in argsCopy:
+            defURL += i
+            defURL += "%20"
+
```

Comment:
Do the following instead to avoid this convoluted chain of if/for statements: 

`tokens = args.split(" ")

 try: 
     defNum = int(tokens[-1])
     defURL += "%20".join(tokens[0:-1])
 except:
     defURL += "%20".join(tokens)
`